test/bdd/features/datalayer.feature

```diff
@@ -76,13 +76,13 @@ Feature: Data layer related features
     And I start additional nodes
     And I use 2nd node as DV
     Given DV publishes query consisting of path: "identifiers.id", value: "urn:epc:id:sgtin:Batch_1" and opcode: "EQ" to the network
-    Then all nodes with last import should answer to last network query by DV
-    Given the DV purchases import from the last query from the DC
+    Then all nodes with lastImport should answer to last network query by DV
```

Comment:
Keep it in domain of natural language. I'd keep 'last import'.
The implementation should be fairly easy i.e.
`(last import|second last import)`

Reply:
agreed, will fix